episodes/severity-static.Rmd

```diff
@@ -0,0 +1,640 @@
+
+```{r}
+# Calculate the static CFR while adjusting for delays
+cfr_static(
+  data = ebola1976,
+  delay_density = function(x) density(onset_to_death_ebola, x)
+)
+```
+
+:::::::::::::::::: callout
+
+### Use the `<epidist>` class
+
+When using an `<epidist>` class object we can use this expression as a template `function(x) density(<EPIDIST_OBJECT>, x)`
+
+For distribution functions with parameters not available in `{epiparameter}`, we suggest you two alternatives: 
+
+- Create an `<epidist>` class object, to plug into other R packages of the outbreak analytics pipeline. Read the [reference documentation of `epiparameter::epidist()`](https://epiverse-trace.github.io/epiparameter/reference/epidist.html), or
+
+- Read `{cfr}` vignette for [a primer on working with delay distributions](https://epiverse-trace.github.io/cfr/articles/delay_distributions.html).
+
+::::::::::::::::::
+
+```{r,echo=FALSE}
+out_delay_adjusted <-
+  cfr_static(
+    data = ebola1976,
+    delay_density = function(x) density(onset_to_death_ebola, x)
+  )
+
+out_mean <- out_delay_adjusted %>% pull(severity_mean)
+out_low <- out_delay_adjusted %>% pull(severity_low)
+out_high <- out_delay_adjusted %>% pull(severity_high)
+```
+
+The delay-adjusted CFR indicated that the overall disease severity _at the end of the outbreak_ or with the _latest data available at the moment_ is `r out_mean` with a 95% confidence interval between `r out_low` and `r out_high`, slightly higher than the naive one.
+
+:::::::::::::::::::::::::::::::::::::::: challenge
+
+Load the `ebola_kikwit_1995` from `{outbreaks}` and estimate the delay-adjusted CFR using the appropriate distribution delay. Then:
+
+- Compare the naive and the delay-adjusted CFR solutions!
+
+:::::::::::::::::::: hint
+
+- Find the appropriate `<epidist>` object!
+
+::::::::::::::::::::
+
+:::::::::::::::::::: solution
+
+```{r}
+library(outbreaks)
+
+outbreaks::ebola_kikwit_1995 %>% glimpse()
+
+outbreaks::ebola_kikwit_1995 %>%
+  rename(
+    cases = onset,
+    deaths = death
+  ) %>%
+  cfr::cfr_static(
+    delay_density = function(x) density(onset_to_death_ebola, x)
+  )
+```
+
+::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::
+
+
+## An early-stage CFR estimate
+
+We just found that the difference between the _naive_ and _delay-adjusted_ CFR estimates using `cfr_static()` are _just slightly_ different. So, what is the main difference?!
+
+The main benefit of using `cfr_static()` is to reduce the time to get an _delay-adjusted CFR_ estimate closer to the overall CFR of the epidemic. 
+
+`cfr_static()` can assess the virulence of an emerging infectious disease at an epidemic's **early stage**. This is particularly useful when an _epidemic curve of confirmed cases_ is the only data available (i.e. when individual data from onset to death are unavailable, especially during the early stage of the epidemic). Nishiura et al., 2009 depict this in the figures below.
```

Comment:
I would not frame it this way. Rather, the static estimate is always useful to get an **overall severity estimate** of the outbreak (so far). Once the outbreak has clearly ended, or has progressed such that deaths are reported more, the estimated CFR is then closest to the 'true' CFR. This is what the figures really show - I would suggest re-interpreting them and modifying this section.

Reply:
I added this rephrase in https://github.com/epiverse-trace/tutorials-middle/pull/1/commits/934a687bbfc0b8eb5b6d70beec25947cd2358525